tap-client/src/components/settings/SettingsComponent.tsx

```diff
@@ -1,10 +1,27 @@
-export default function SettingsComponent() {
-    return (
-        <div className="flex justify-center items-center h-screen p-4">
-            <div className="text-center">
-                <h1 className="text-3xl mb-4">Settings</h1>
-                <p className="text-lg">Update your settings here.</p>
-            </div>
-        </div>
-    );
-}
+
+import Collapse from "./CollapseComponent";
+import CollapseInfo from "./CollapseInfoComponent";
+
+function Settings() {
+    //ToDO - Get Data pass it into a collpaseInfo Comp with title and ex. data={PersonalInfo.fname}
+  return (
+    <div className="h-screen flex flex-col items-center justify-center">
+      <Collapse title="Personal Information">
+        <CollapseInfo title="First Name" data={""}/>
+        <CollapseInfo title="Last Name"/>
+        <CollapseInfo title="Email"/>
+        <CollapseInfo title="Bachelor's Degree"/>
+        <CollapseInfo title="Major"/>
+        <CollapseInfo title="Expected Graduation"/>
+        <CollapseInfo title="Honors College?" />
+      </Collapse>
+      <Collapse title="Preferences">
+        <CollapseInfo title="Preferred Credit Hours" data={""}/>
+        <CollapseInfo title="Unavailable Terms"/>
+        <CollapseInfo title="Additional Preferences"/>
+      </Collapse>
```

Comment:
Create a for loop and create these elements based off the data in preferencesInfo